### DISCUSSION TITLE
The scope of this project

### DISCUSSION THREAD

**Discussion by bluelightzero** [lawrencehook/remove-youtube-suggestions#103] [General]
This isn't so much an idea, but more a question of where this project is going and where the scope of features starts and ends.

The description of this plugin is:

> This extension aims to make YouTube less engaging and more configurable. It provides options to hide recommended videos and to customize the user interface.

I mostly use this plugin because I have ADHD and it has allowed me to still use youtube without becoming too engaged.

I also use it however because I am upset with direction youtube has taken with UI and it has given me more control in hiding features that I have no intent to use such as shorts.

I was wondering if there were plans to make this a larger scope youtube plugin in the same vein as RES for reddit? Maybe in a way that can completely overhaul entire features such as how subscriptions and playlists work.

One feature I would really want is a completely local data solution for youtube so I can still use the whole website without even being logged in. Things like keeping a watch history of where in a video or playlist I am up to (Very unreliable on youtube's end) managing separate subscription feeds, a more responsive UI (Youtube is horrendously slow sometimes). Youtube makes it very hard to tell which video was removed from a playlist so keeping a backup of video meta data so I can find lost videos, and be more informed.

Obviously this is far out of the scope of simply hiding suggestions, I want to hear your thoughts.

**Comment by lawrencehook** [comment]
Hello! Thanks for making this conversation.

I'm certainly open to what you are describing. I'm a fan of RES! However, it'd likely require a lot of effort. I work on this project only occasionally.
